src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml

```diff
@@ -769,6 +770,17 @@ services:
 
   PrestaShopBundle\Form\Admin\AdvancedParameters\AuthorizationServer\ApplicationType:
   PrestaShopBundle\Form\Admin\Improve\International\Tax\TaxRulesGroupType:
+  PrestaShopBundle\Form\Admin\Sell\CartRule\CartRuleType:
+  PrestaShopBundle\Form\Admin\Sell\CartRule\ActionsType:
+  PrestaShopBundle\Form\Admin\Sell\CartRule\ConditionsType:
+  PrestaShopBundle\Form\Admin\Sell\CartRule\InformationType:
+  PrestaShopBundle\Form\Admin\Sell\CartRule\MinimumAmountType:
+  PrestaShopBundle\Form\Admin\Sell\CartRule\DiscountType:
```

Comment:
can you group them to include all files into the same directory

Reply:
sorry I don't follow, what do you mean? files already are in same directory :thinking:

Reply:
Sorry I was not clear enough, You can do like I did for form extensions: 

You can group them, at least for the cart rules form, so we don't have to think anymore about declaring them. This is just an example, I did not tested it.

```yaml
  PrestaShopBundle\Form\Admin\Sell\CartRule\:
    resource: "../../../../Form/Admin\Sell\CartRule\/*"
```

Reply:
ahh nice :+1:

Reply:
much cleaner this way, sadly phpstorm doesn't see the definitions and cannot navigate in editor, that's a bummer :(